test/acvp_test.c

```diff
@@ -100,8 +102,9 @@ static int sig_gen(EVP_PKEY *pkey, OSSL_PARAM *params, const char *digest_name,
     sig_len = sz;
     if (!TEST_ptr(sig = OPENSSL_malloc(sz))
         || !TEST_ptr(md_ctx = EVP_MD_CTX_new())
-        || !TEST_int_eq(EVP_DigestSignInit_ex(md_ctx, NULL, digest_name, libctx,
-                                              NULL, pkey, NULL), 1)
+        || !TEST_int_eq(EVP_DigestSignInit_ex(md_ctx, &pkey_ctx, digest_name,
+                                              libctx, NULL, pkey, NULL), 1)
```

Comment:
you should also be able to pass the params here as the last param to DigestSignInit_ex()

Reply:
Updated. The `params` is passed to context via `EVP_DigestSignInit_ex`.